src/controller/scan/base_controller.go

```diff
@@ -600,7 +600,7 @@ func (bc *basicController) makeRobotAccount(ctx context.Context, projectID int64
 			Name:        fmt.Sprintf("%s-%s", registration.Name, UUID),
 			Description: "for scan",
 			ProjectID:   projectID,
-			ExpiresAt:   -1,
+			Duration:    int64(config.RobotTokenDuration()),
```

Comment:
This line can be removed as the robot controller will set it to the value from the configuration when it's zero.

Reply:
removed